egui_demo_lib/src/apps/demo/table_demo.rs

```diff
@@ -28,14 +30,30 @@ impl super::Demo for TableDemo {
 
 impl super::View for TableDemo {
     fn ui(&mut self, ui: &mut egui::Ui) {
-        ui.checkbox(&mut self.virtual_scroll, "Virtual scroll");
-        ui.checkbox(&mut self.resizable, "Resizable columns");
-
         // Leave room for the source code link after the table demo:
         StripBuilder::new(ui)
+            .size(Size::exact(66.0)) // for the settings
             .size(Size::remainder()) // for the table
             .size(Size::exact(10.0)) // for the source code link
             .vertical(|mut strip| {
+                strip.cell(|ui| {
+                    StripBuilder::new(ui)
+                        .size(Size::exact(150.0))
+                        .size(Size::remainder())
+                        .horizontal(|mut strip| {
+                            strip.cell(|ui| {
+                                ui.checkbox(&mut self.heterogeneous_rows, "Heterogeneous rows");
+                                ui.checkbox(&mut self.virtual_scroll, "Virtual Scroll");
```

Comment:
These are 2*2=4 options for three different demos. Perhaps use an enum and radio buttons, or only show `heterogeneous_rows` in case `virtual_scroll==true`

Reply:
I took the approach of only showing the `heterogeneous_rows` setting if `virtual_scroll==true`.